server/client.go

```diff
@@ -709,6 +725,7 @@ func (c *client) deliverMsg(sub *subscription, mh, msg []byte) {
 
 	deadlineSet := false
 	if client.bw.Available() < (len(mh) + len(msg) + len(CR_LF)) {
+		atomic.AddInt64(&c.cache.wfc, 1)
```

Comment:
No you are right, these are for the clients that we have pending flushes for, outbound, so I am checking wrong client, will fix.